src/App.js

```diff
@@ -15,49 +15,46 @@ import SearchForm from './components/SearchForm';
 import ReleaseDateToggle from './components/ReleaseDateToggle';
 import MyModal from './components/Modal';
 import Form from './components/Form';
+import HandleSearch from './helper/utils/handleSearch';
 
 import filmsData from './filmsData';
 import './App.scss';
+import { SORT_CONST } from './helper/constants';
 
 const App = () => {
-
-  const handleSearch = (search) => {
-    console.log(search);
-  }
-
   const [modalShow, setModalShow] = useState(false);
+  const [sortValue, setSortValue] = useState(null);
+  const [filteredData, setFilteredData] = useState(filmsData || []);
 
-  const [data, setNewData] = useState(filmsData);
-  // const [filteredData, setFilteredData] = useState(filmsData);
-
-  
-  const sortReleaseDate = (e) => {
-
-    const sortData = data.map(item => item).sort((a, b) => {
-      if(e.value === 'down to') {
+  const sortReleaseDate = (data) => {
+    return data.map(item => item).sort((a, b) => {
```

Comment:
check please if you really  need map here

Reply:
yes, delete it